index.rst

```diff
@@ -429,6 +429,7 @@ Output Components
     SM2135, components/output/sm2135, sm2135.svg
     MCP4725, components/output/mcp4725, mcp4725.jpg
     MCP47A1, components/output/mcp47a1, mcp47a1.svg
+    MCP4728, components/output/mcp4728, mcp4728.jpg
```

Comment:
The jpg is rather big (we try to really minimize the files because they're loaded by every user when opening the index page). Here's an optimized version created by https://compress-or-die.com/
![mcp4728(1)](https://user-images.githubusercontent.com/6833237/154756879-dca284f9-ee5b-4f95-90c2-a2d3df04c22b.jpg)